src/main/src/main/java/org/geoserver/security/urlchecks/AbstractURLCheck.java

```diff
@@ -43,20 +43,24 @@ public void setDescription(String description) {
         this.description = description;
     }
 
-    /** @return the enable */
+    /** @return the enabled status */
     @Override
-    public boolean isEnabled() {
+    public Boolean isEnabled() {
         return enabled;
     }
 
-    /** @param enabled the enable to set */
-    public void setEnabled(boolean enabled) {
+    /** @param enabled the enabled status to set */
+    public void setEnabled(Boolean enabled) {
         this.enabled = enabled;
     }
 
+    /** @return the configuration of this checker */
+    public abstract String getConfiguration();
+
     /**
-     * A short string representation of the configuration of this checker, to be used for generic
-     * display purposes
```

Comment:
Why was the javadoc for the getter removed and replaced with a less useful statement?

Reply:
Because with these changes, the configuration is no longer used for generic display purposes. The previous Javadoc might have been less relevant in the context of the updated functionality.

Reply:
Reverted. Thanks.